examples/bri-3/src/bri/auth/guards/didJwt.guard.ts

```diff
@@ -44,7 +44,7 @@ export class DidJwtAuthGuard implements CanActivate {
     const didSubstrLength = 13;
     const bpiSubject =
       await this.bpiSubjectStorageAgent.getBpiSubjectByPublicKey(
-        verified.payload.sub.substring(didSubstrLength),
+        verified.payload.sub!.substring(didSubstrLength),
```

Comment:
'!` ??? @ognjenkurtic

Reply:
We switched to structNullChecks in the wholecodebase and typescript complains about a possible null value for .sub here. Since it cannot be undefined,  i added the[ bang operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) to tell it.